packages/react-components/react-tabster/package.json

```diff
@@ -36,8 +36,8 @@
     "@fluentui/react-utilities": "^9.15.6",
     "@griffel/react": "^1.5.14",
     "@swc/helpers": "^0.5.1",
-    "keyborg": "^2.3.0",
-    "tabster": "^5.0.1"
+    "keyborg": "2.5.0-canary.0",
```

Comment:
using non stable versions in stable v9 core packages is not something we should do

Reply:
Agreed. I don't think the plan is to actually merge the unstable versions but rather to spin up the PR with the changes that should land in these deps soon.